addons/medical/functions/fnc_vitalLoop.sqf

```diff
@@ -34,6 +34,24 @@ if (!local _unit) exitWith {
 // Handle unit vitals
 [_unit, CBA_missionTime - _lastTime] call FUNC(handleUnitVitals);
 
+// remove old actions
+[_unit] call FUNC(clearExpired);
+
+// Check damage or bleeing for Splint
+private _damage = _unit getVariable [QGVAR(bodyPartStatus), [0, 0, 0, 0, 0, 0]];
+private _tourniquets = _unit getVariable [QGVAR(tourniquets), [0, 0, 0, 0, 0, 0]];
+private _openWounds = _unit getVariable [QGVAR(openWounds), []];
+private _splint = _unit getVariable [QGVAR(samSplint), [[0, 0], [0, 0], [0, 0], [0, 0], [0, 0], [0, 0]]];
+for "_i" from 0 to 5 do {
+    private _hasDamage = (_damage select _i) > 0;
+    private _hasTourniquet = (_tourniquets select _part) != 0;
+    private _bleedingWound = {((_x select 2) isEqualTo _part) && ((_x select 4) * (_x select 3) > 0)} count ([_openWounds, _bandagedWounds] select _bandagedAlso) > 0;
```

Comment:
The amount of `_x select #` here make it difficult to follow what's happening. I'd abstract those indexes away into defines. We may have done that in some parts of the code already, irc. If not, we should do it here and in a follow up PR the rest of them.

Reply:
AFAIK codes related to wounds are done similarly, for ex) https://github.com/acemod/ACE3/blob/master/addons/medical/functions/fnc_getBloodLoss.sqf#L29
It's just how wound array works, and I thought declaring each items as separate variables will not be a good idea in the view of optimization.

Reply:
>and I thought declaring each items as separate variables will not be a good idea in the view of optimization.

@thojkooi was talking about Macros. Not variables.
So `_x select OFFSET_WOUNDS_PART` instead of `_x select 2`. More readable and 0 performance difference.

Reply:
Ok, will do that ASAP